specs/core/0_beacon-chain.md

```diff
@@ -219,6 +220,9 @@ These configurations are updated for releases, but may be out of sync during `de
 | `RANDAO_MIXES_LENGTH` | `2**13` (= 8,192) | epochs | ~36 days |
 | `ACTIVE_INDEX_ROOTS_LENGTH` | `2**13` (= 8,192) | epochs | ~36 days |
 | `SLASHED_EXIT_LENGTH` | `2**13` (= 8,192) | epochs | ~36 days |
+| `VALIDATOR_REGISTRY_SIZE` | `2**40 (= 1,099,511,627,776)` | | |
+
+* Assuming the maximum 16 deposits per slot, the validator registry will last for at least 13,065 years (but likely much much longer)
```

Comment:
I'm tempted to remove this comment for a couple reasons:

1) In phase 2 it may be possible to create more than `MAX_DEPOSITS = 16` new validators per slot.
2) We don't usually put design rationales in the spec. (A separate design rationale doc needs to be created.)